templates/lookup/ifupdown__dynamic_packages.j2

```diff
@@ -16,6 +16,4 @@
 {%     endif %}
 {%   endfor %}
 {% endif %}
-{% if ifupdown__tpl_packages %}
 {{ ifupdown__tpl_packages | unique | to_yaml }}
-{% endif %}
```

Comment:
The `bridge-utils` package is added conditionally. I meant a case where no conditional packages are present, ie. `ifupdown__dynamic_packages` ends up being empty. Try it with a host which only has Ethernet interfaces, no bridges, vlans, etc.